map/src/data/markers.ts

```diff
@@ -62,6 +62,32 @@ export interface MarkerInfo {
 }
 
 const LOCATIONS = {
+  PT: {
+    ALMADA: {
+      description: 'Almada, Portugal',
+      lat: 38.678506,
+      lng: -9.164985,
+      serviceRadius: 5000,
+    },
+    LISBON_BELEM: {
+      description: 'Lisbon, Belém, Portugal',
+      lat: 38.7026973,
+      lng: -9.2089478,
+      serviceRadius: 2000,
+    },
+    PORTO_FOZ_DO_DOURO: {
+      description: 'Porto, Foz do Douro, Portugal',
+      lat: 41.1675792,
+      lng: -8.7027557,
+      serviceRadius: 2000,
+    },
+    ALL: {
+      description: 'PT',
+      lat: 54.200981,
+      lng: -4.513111,
+      serviceRadius: 280610,
```

Comment:
It looks like these are the coordinates for all of the UK not Portugal

Reply:
Crap, sorry -.-

Reply:
Fixed here - https://github.com/reach4help/reach4help/pull/369/commits/e3f0027fcc01b97173335afe6eb2e83c35650463